packages/coil-extension/src/popup/mocks/loadMockedStates.tsx

```diff
@@ -146,7 +146,8 @@ const userCanTip = mockState({
 
 const userCanTipNewUi = mockState({
   monetized: true,
-  coilSite: 'https://coil.com/p/mirrae/Letter-to-my-Daughter/-ZInTJqN-',
```

Comment:
Same here

Reply:
This was intentional, `loadMockedState` is the mocked data for the view previewer. That's essentially what I use to build the views.